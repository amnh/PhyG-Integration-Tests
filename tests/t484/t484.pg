--fuse(all, unique)), inputting prefasta file
set(seed:1634561640)
read(prefasta: "chel-prealigned.fas")
read(prefasta: "metazoa-aa-1.fas")
build(distance, rdWag, block, displaytrees:10, atrandom)
fuse(all, unique)
report("t484.csv", data, overwrite)
report("t484.tre", newick, overwrite)