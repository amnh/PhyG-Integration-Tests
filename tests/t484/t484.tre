(Alentus:0.0,(Mytilus_californianus:0.0,(Venerupis_philippinarum:10.0,(Yoldia_limatula:5.0,(Aportus:8.0,(Thermobius:18.0,(((Mastigoproctus:7.0,(((((Aplysia_californica:1.0,(Gea:0.0,Siphonaria_pectinata:0.0)HTU46:1.0)HTU45:13.0,(Thelichoris:3.0,(Crassostrea_virginica:0.0,Hypochilus:0.0)HTU48:0.0)HTU47:13.0)HTU44:11.0,(((Littorina_littorea:2.0,(Hadrurus:0.0,Lottia_gigantea:0.0)HTU52:0.0)HTU51:5.0,(Pauroctonus:0.0,Scaphopods:0.0)HTU53:2.0)HTU50:0.0,(Ilyanassa_obsoleta:1.0,Perotrochus_lucaya:0.0)HTU54:1.0)HTU49:4.0)HTU43:2.0,(Centruoides:0.0,Helobdella_robusta:0.0)HTU55:4.0)HTU42:0.0,(Chanbria:11.0,Limulus:9.0)HTU56:4.0)HTU41:1.0)HTU40:6.0,(Amblypygid:0.0,Lymnaea_stagnalis:0.0)HTU57:3.0)HTU39:13.0,((Vonones:14.0,(Americhernus:0.0,Venerupis_decussatus:0.0)HTU60:15.0)HTU59:26.0,(Artemia:34.0,(Lingula_anatina:0.0,Rhiphicephalus:0.0)HTU62:7.0)HTU61:6.0)HTU58:13.0)HTU38:2.0)HTU37:21.0)HTU36:0.0)HTU35:8.0)HTU34:0.0)HTU33:0.0)HTU32[624.0];
