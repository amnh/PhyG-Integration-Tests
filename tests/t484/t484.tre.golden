(Alentus:70.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU33:8.5,(Artemia:72.0,Scaphopods:79.0)HTU34:23.0)HTU35:17.5,(Thermobius:78.5,((Amblypygid:68.0,(Mastigoproctus:66.5,((Centruoides:68.5,(Crassostrea_virginica:113.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU52:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU50:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU51:17.0)HTU53:10.0)HTU54:3.5)HTU55:7.5)HTU56:8.5)HTU57:11.0,(Lottia_gigantea:80.0,(((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,(Chanbria:59.5,Venerupis_philippinarum:80.5)HTU42:4.0)HTU43:6.5)HTU44:7.0)HTU45:34.5,(Limulus:62.0,(Ilyanassa_obsoleta:77.5,Littorina_littorea:76.5)HTU41:2.5)HTU46:6.0)HTU47:8.5,(Gea:85.0,(Perotrochus_lucaya:74.5,(Hypochilus:60.0,Thelichoris:62.0)HTU39:3.5)HTU40:3.5)HTU48:10.0)HTU49:4.5)HTU58:4.5)HTU59:10.0)HTU60:9.0)HTU61:12.0,(Aportus:88.0,Vonones:70.0)HTU62:6.5)HTU37:10.0)HTU38:5.5)HTU36:12.5)HTU32[615.0];
(Alentus:70.5,(Thermobius:78.0,(((Americhernus:99.5,Rhiphicephalus:75.5)HTU33:8.5,(Artemia:72.0,Scaphopods:79.0)HTU34:23.0)HTU35:16.5,((Amblypygid:67.5,(Mastigoproctus:66.5,((Centruoides:68.5,(Crassostrea_virginica:113.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU52:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU50:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU51:17.0)HTU53:10.0)HTU54:3.5)HTU55:7.5)HTU56:8.5)HTU57:11.0,(Lottia_gigantea:80.0,(((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,(Chanbria:59.5,Venerupis_philippinarum:80.5)HTU42:4.0)HTU43:6.5)HTU44:7.0)HTU45:34.5,(Limulus:62.0,(Ilyanassa_obsoleta:77.5,Littorina_littorea:76.5)HTU41:2.5)HTU46:6.0)HTU47:8.5,(Gea:85.0,(Perotrochus_lucaya:74.5,(Hypochilus:60.0,Thelichoris:62.0)HTU39:3.5)HTU40:3.5)HTU48:10.0)HTU49:4.5)HTU58:4.5)HTU59:10.0)HTU60:9.5)HTU61:11.0,(Aportus:88.0,Vonones:70.0)HTU62:7.5)HTU37:9.5)HTU38:6.0)HTU36:13.0)HTU32[615.0];
(Alentus:69.0,(Thermobius:80.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU33:8.0,(Artemia:71.5,Scaphopods:79.0)HTU34:24.0)HTU35:17.5,(Aportus:83.5,(Vonones:67.5,(Amblypygid:67.5,(Mastigoproctus:66.5,((Centruoides:68.5,(Crassostrea_virginica:113.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU52:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU50:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU51:17.0)HTU53:10.0)HTU54:3.5)HTU55:7.5)HTU56:8.5)HTU57:11.0,(Lottia_gigantea:80.0,(((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,(Chanbria:59.5,Venerupis_philippinarum:80.5)HTU42:4.0)HTU43:6.5)HTU44:7.0)HTU45:34.5,(Limulus:62.0,(Ilyanassa_obsoleta:77.5,Littorina_littorea:76.5)HTU41:2.5)HTU46:6.0)HTU47:8.5,(Gea:85.0,(Perotrochus_lucaya:74.5,(Hypochilus:60.0,Thelichoris:62.0)HTU39:3.5)HTU40:3.5)HTU48:10.0)HTU49:4.5)HTU58:4.5)HTU59:10.0)HTU60:9.5)HTU61:10.5)HTU62:13.0)HTU37:6.5)HTU38:3.5)HTU36:12.0)HTU32[608.0];
(Alentus:69.0,(Thermobius:80.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU33:8.0,(Artemia:71.5,Scaphopods:79.0)HTU34:24.0)HTU35:17.5,(Aportus:83.5,(Vonones:67.5,(Amblypygid:67.5,(Mastigoproctus:66.5,((Centruoides:68.5,(Crassostrea_virginica:113.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU52:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU50:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU51:17.0)HTU53:10.0)HTU54:3.5)HTU55:7.5)HTU56:8.5)HTU57:11.0,(Lottia_gigantea:80.0,(Perotrochus_lucaya:75.0,(((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,(Chanbria:59.5,Venerupis_philippinarum:80.5)HTU41:4.0)HTU42:6.5)HTU43:7.0)HTU44:34.5,(Limulus:62.0,(Ilyanassa_obsoleta:77.5,Littorina_littorea:76.5)HTU40:2.5)HTU45:6.0)HTU46:8.5,(Gea:85.0,(Hypochilus:60.0,Thelichoris:62.0)HTU39:4.5)HTU47:10.0)HTU48:3.5)HTU49:3.5)HTU58:4.5)HTU59:10.0)HTU60:9.5)HTU61:10.5)HTU62:13.0)HTU37:6.5)HTU38:3.5)HTU36:12.0)HTU32[608.0];
(Alentus:69.0,(Thermobius:80.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU54:8.0,(Artemia:71.5,Scaphopods:79.0)HTU55:24.0)HTU56:17.5,(Aportus:83.5,(Vonones:67.5,(Amblypygid:67.5,(Mastigoproctus:66.5,((Centruoides:68.5,(Crassostrea_virginica:113.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU43:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU41:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU42:17.0)HTU44:10.0)HTU45:3.5)HTU46:7.5)HTU47:8.5)HTU48:11.0,(Lottia_gigantea:80.0,((Gea:85.0,(Perotrochus_lucaya:74.5,(Hypochilus:60.0,Thelichoris:62.0)HTU33:3.5)HTU34:3.5)HTU39:10.0,((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,(Chanbria:59.5,Venerupis_philippinarum:80.5)HTU35:4.0)HTU36:6.5)HTU37:7.0)HTU38:34.5,(Ilyanassa_obsoleta:77.5,(Limulus:62.0,Littorina_littorea:76.0)HTU62:3.0)HTU60:6.0)HTU61:8.5)HTU40:4.5)HTU49:4.5)HTU50:10.0)HTU51:9.5)HTU52:10.5)HTU53:13.0)HTU57:6.5)HTU58:3.5)HTU59:12.0)HTU32[608.0];
(Alentus:69.0,(Thermobius:80.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU54:8.0,(Artemia:71.5,Scaphopods:79.0)HTU55:24.0)HTU56:17.5,(Aportus:83.5,(Vonones:67.5,(Amblypygid:67.5,(Mastigoproctus:66.5,((Centruoides:68.5,(Crassostrea_virginica:113.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU43:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU41:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU42:17.0)HTU44:10.0)HTU45:3.5)HTU46:7.5)HTU47:8.5)HTU48:11.0,(Lottia_gigantea:80.0,((Gea:85.0,(Perotrochus_lucaya:74.5,(Hypochilus:60.0,Thelichoris:62.0)HTU33:3.5)HTU34:3.5)HTU39:10.0,((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,(Chanbria:59.5,Venerupis_philippinarum:80.5)HTU35:4.0)HTU36:6.5)HTU37:7.0)HTU38:34.5,(Littorina_littorea:76.5,(Ilyanassa_obsoleta:76.5,Limulus:62.0)HTU62:3.5)HTU60:6.0)HTU61:8.5)HTU40:4.5)HTU49:4.5)HTU50:10.0)HTU51:9.5)HTU52:10.5)HTU53:13.0)HTU57:6.5)HTU58:3.5)HTU59:12.0)HTU32[608.0];
(Alentus:69.0,(Thermobius:80.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU54:8.0,(Artemia:71.5,Scaphopods:79.0)HTU55:24.0)HTU56:17.5,(Aportus:83.5,(Vonones:67.5,(Amblypygid:67.5,(Mastigoproctus:66.5,((Centruoides:68.5,(Crassostrea_virginica:113.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU43:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU41:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU42:17.0)HTU44:10.0)HTU45:3.5)HTU46:7.5)HTU47:8.5)HTU48:11.0,(Lottia_gigantea:80.0,(Perotrochus_lucaya:75.0,(((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Chanbria:59.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU62:4.0)HTU60:5.0)HTU61:7.0)HTU35:34.5,(Limulus:62.0,(Ilyanassa_obsoleta:77.5,Littorina_littorea:76.5)HTU34:2.5)HTU36:6.0)HTU37:8.5,(Gea:85.0,(Hypochilus:60.0,Thelichoris:62.0)HTU33:4.5)HTU38:10.0)HTU39:3.5)HTU40:3.5)HTU49:4.5)HTU50:10.0)HTU51:9.5)HTU52:10.5)HTU53:13.0)HTU57:6.5)HTU58:3.5)HTU59:12.0)HTU32[608.0];
(Alentus:69.0,(Thermobius:80.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU54:8.0,(Artemia:71.5,Scaphopods:79.0)HTU55:24.0)HTU56:17.5,(Aportus:83.5,(Vonones:67.5,(Amblypygid:67.5,(Mastigoproctus:66.5,((Centruoides:68.5,(Crassostrea_virginica:113.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU43:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU41:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU42:17.0)HTU44:10.0)HTU45:3.5)HTU46:7.5)HTU47:8.5)HTU48:11.0,(Lottia_gigantea:80.0,(Perotrochus_lucaya:75.0,(((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_philippinarum:82.0,(Chanbria:59.5,Venerupis_decussatus:81.0)HTU62:31.0)HTU60:6.5)HTU61:7.0)HTU35:34.5,(Limulus:62.0,(Ilyanassa_obsoleta:77.5,Littorina_littorea:76.5)HTU34:2.5)HTU36:6.0)HTU37:8.5,(Gea:85.0,(Hypochilus:60.0,Thelichoris:62.0)HTU33:4.5)HTU38:10.0)HTU39:3.5)HTU40:3.5)HTU49:4.5)HTU50:10.0)HTU51:9.5)HTU52:10.5)HTU53:13.0)HTU57:6.5)HTU58:3.5)HTU59:12.0)HTU32[608.0];
(Alentus:69.0,(Thermobius:80.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU48:8.0,(Artemia:71.5,Scaphopods:79.0)HTU49:24.0)HTU50:17.5,(Aportus:83.5,(Vonones:67.5,(Amblypygid:67.5,(Mastigoproctus:66.5,((Lottia_gigantea:80.0,(Perotrochus_lucaya:75.0,(((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,(Chanbria:59.5,Venerupis_philippinarum:80.5)HTU35:4.0)HTU36:6.5)HTU37:7.0)HTU38:34.5,(Limulus:62.0,(Ilyanassa_obsoleta:77.5,Littorina_littorea:76.5)HTU34:2.5)HTU39:6.0)HTU40:8.5,(Gea:85.0,(Hypochilus:60.0,Thelichoris:62.0)HTU33:4.5)HTU41:10.0)HTU42:3.5)HTU43:3.5)HTU44:4.5,(Crassostrea_virginica:113.5,(Centruoides:68.5,(Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU58:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU56:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU57:17.0)HTU59:10.0)HTU60:3.5)HTU61:12.5)HTU62:6.0)HTU54:8.5)HTU55:10.0)HTU45:9.5)HTU46:10.5)HTU47:13.0)HTU51:6.5)HTU52:3.5)HTU53:12.0)HTU32[608.0];
(Alentus:69.0,(Thermobius:80.5,(((Americhernus:98.5,Rhiphicephalus:76.5)HTU48:8.0,(Artemia:71.5,Scaphopods:79.0)HTU49:24.0)HTU50:17.5,(Aportus:83.5,(Vonones:67.5,(Amblypygid:67.5,(Mastigoproctus:66.5,((Lottia_gigantea:80.0,(Perotrochus_lucaya:75.0,(((Mytilus_californianus:84.5,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,(Chanbria:59.5,Venerupis_philippinarum:80.5)HTU35:4.0)HTU36:6.5)HTU37:7.0)HTU38:34.5,(Limulus:62.0,(Ilyanassa_obsoleta:77.5,Littorina_littorea:76.5)HTU34:2.5)HTU39:6.0)HTU40:8.5,(Gea:85.0,(Hypochilus:60.0,Thelichoris:62.0)HTU33:4.5)HTU41:10.0)HTU42:3.5)HTU43:3.5)HTU44:4.5,((Hadrurus:69.5,((Helobdella_robusta:90.5,Yoldia_limatula:84.0)HTU58:3.5,((Lymnaea_stagnalis:77.0,Pauroctonus:59.5)HTU56:2.5,(Aplysia_californica:85.5,Siphonaria_pectinata:86.5)HTU57:17.0)HTU59:10.0)HTU60:3.5)HTU61:12.5,(Centruoides:63.0,Crassostrea_virginica:88.5)HTU62:32.5)HTU54:12.0)HTU55:10.0)HTU45:9.5)HTU46:10.5)HTU47:13.0)HTU51:6.5)HTU52:3.5)HTU53:12.0)HTU32[608.0];
